src/main/java/frc/team2412/robot/subsystems/DrivebaseSubsystem.java

```diff
@@ -334,7 +334,7 @@ private void initShuffleboard() {
 						.getEntry();
 		xWheelsEnabled = xWheelsEntry.getBoolean(true);
 		flipTranslationEntry =
-				drivebaseTab
+				Shuffleboard.getTab("Match")
```

Comment:
Is this something that needs to be on the Match tab? On the one hand, we do want DT to be able to use it quickly when they have to, but they also shouldn't have to use it.

Reply:
probably dont need it anymore since the translation issue durring sammy is fixed, i guess ill put it in drivebase for now